src/kirby/components/chart/options/areaspline.ts

```diff
@@ -10,6 +10,9 @@ export const AreaSplineOptions: Options = {
     margin: 0,
     marginTop: 48,
   },
+  accessibility: {
```

Comment:
This looks bad (for accessibility)... won't people with sight disabilities have a problem figuring out what the graph is (based on this option)?